Document the getting started steps for multi-language (#564)
seek-oss/sku: docs/docs/multi-language.md

**Note:** sku assumes that the development language is `en`. This will be used as the base language for all development, including your initial translations.

## Getting started

> To opt-in to Multi-language builds you need to perform three steps:

Reviewer: ```suggestion
To opt-in to multi-language builds you need to perform three steps:
```

1. Install "`@vocab/react`" as a dependency.
   This will be used by your code to support dynamically loading the correct language translations inside React components

> 2. Add "`languages`" attribute to sku.config with all your supported languages

Reviewer: ```suggestion
2. Add the list of supported "`languages`" to your [sku configuration](./docs/configuration.md#languages)
```
